website/html/views.py

```diff
@@ -868,7 +873,9 @@ def priv_key(request):
   except LoggedInButFailedGetGeniUserError:
     return _show_failed_get_geniuser_page(request)
   
-  response = HttpResponse(user.user_privkey, mimetype='text/plain')
+  # Fixed for Issue #152, compatibility with Django 1.8+ requires shift from
+  # mime_type argument to content_type argument.

```

Comment:
[True](https://docs.djangoproject.com/en/dev/internals/deprecation/#deprecation-removed-in-1-7)